### ISSUE TITLE
【Windows】 process.Percent > 100

### ISSUE THREAD

**Issue by Nyx2022** [shirou/gopsutil#1752]
**Describe the bug**
in windows, process.Percent > 100

![image](https://github.com/user-attachments/assets/e4a31c32-4db0-44de-a475-ccb72d1e4373)

**Expected behavior**
get the value show in explorer, may be a litlle difference

**Environment (please complete the following information):**
 - [ ] Windows: [Microsoft Windows [ver 10.0.26100.2454]]

**Additional context**

**Comment by little-nil** [comment]
cpuPercentProcess, _ := proc.Percent(0)
cpuCount, _ := cpu.Counts(false)
cpuPercentProcess / float64(cpuCount)

**Comment by hitzhangjie** [comment]
Yes, you should dive the number of cpus to get the value of single cpu
